Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Legolas"
 uuid = "741b9549-f6ed-4911-9fbf-4a1c0c97f0cd"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.5.8"
+version = "0.5.9"
```

Comment:
non-breaking: existing functions weren't removed; user-facing change is body of error messages, although they are still the same type of error (`ArgumentError`).